### PR TITLE
Exposed dataset on website

### LINKED ISSUE #534: Expose dataset on website
- [x] I've checked that this isn't a duplicate issue.

Expose json dataset on the website, and add a link to it.
Url can look like: `https://devswag.io/data.json`

### PR DESCRIPTION
**Pull request by ARKEOLOGIST**
- [X] I've checked that this isn't a new swag opportunity proposal.
- [X] I've checked that this isn't a duplicate pull request.

Made a few minor changes to fix #534
The dataset can now be viewed on-site through the "view data" hyperlink at the bottom of the page.

### CODE REVIEW

**Comment by ARKEOLOGIST** [comment]
Hi, is it possible to get feedback on this?

**Comment by ARKEOLOGIST** [comment]
@aslafy-z can you have a look at this please?

**Comment by aslafy-z** [comment]
So sorry, I missed this PR. I'd go with a netlify redirect to GitHub raw, instead of hosting the JSON directly within the website. What do you think @vikaspotluri123?

**Comment by vikaspotluri123** [comment]
I agree, can just link to `https://github.com/swapagarwal/swag-for-dev/blob/master/data.json`

**Comment by ARKEOLOGIST** [comment]
Done the needful.

File: gulpfile.js [hunk 1]
```diff
@@ -85,6 +85,11 @@ gulp.task('pug', async done => {
 		.on('end', () => done());
 });
 
+gulp.task('data', () => {
```
Comment (aslafy-z) [inline gulpfile.js:88]: Please revert these changes

**Review by aslafy-z — CHANGES_REQUESTED** [review]

**Comment by ARKEOLOGIST** [comment]
Done.

File: gulpfile.js (hunk 1 above)
Comment (ARKEOLOGIST) [reply gulpfile.js]: Done.

**Comment by aslafy-z** [comment]
Perfect, please rebase on master. Then we will have to rehost Accord project image url to be able to see a preview.

**Comment by ARKEOLOGIST** [comment]
You might have to rehost the Accord project image URL since it's the one causing the issue in the build.

**Comment by vikaspotluri123** [comment]
If you rebase against our master, the build should pass since I added support for failing images in PRs. This allows us to preview and fix the swag image separately

**Pull request closed without merging**